---
title: C6322 | Dokumentace Microsoftu
ms.custom: ''
ms.date: 2018-06-30
ms.prod: visual-studio-dev14
ms.reviewer: ''
ms.suite: ''
ms.technology:
- vs-devops-test
ms.tgt_pltfrm: ''
ms.topic: article
f1_keywords:
- C6322
helpviewer_keywords:
- C6322
ms.assetid: fb23d2b1-b2a0-465c-8bf5-ec039c6c7757
caps.latest.revision: 15
author: corob-msft
ms.author: gewarren
manager: ghogen
ms.openlocfilehash: 2c96017e0090dec17a2bcccbed72620da99fe0b5
ms.sourcegitcommit: 55f7ce2d5d2e458e35c45787f1935b237ee5c9f8
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 08/22/2018
ms.locfileid: "42669903"
---
# <a name="c6322"></a>C6322
[!INCLUDE[vs2017banner](../includes/vs2017banner.md)]

Nejnovější verzi tohoto tématu můžete najít v [C6322](https://docs.microsoft.com/visualstudio/code-quality/c6322).  
  
upozornění C6322: prázdný blok _except  
  
 Tato zpráva znamená, blok _except není žádný kód. V důsledku toho je možné dát neošetřené výjimky.  
  
## <a name="example"></a>Příklad  
 Následující kód vygeneruje toto upozornění:  
  
```  
#include <stdio.h>  
#include <excpt.h>  
#include <windows.h>  
  
void fd(char *pch)  
{  
   __try  
     {  
       // exception rasied if pch is null  
       *pch= 0 ;  
     }  
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION)  
     {  
     }  
}  
```  
  
 Chcete-li opravit toto upozornění, použijte následující kód:  
  
```  
#include <stdio.h>  
#include <excpt.h>  
#include <windows.h>  
  
void f(char *pch)  
{  
   __try  
     {  
       // exception rasied if pch is null  
      *pch= 0 ;  
     }  
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION ?   
               EXCEPTION_EXECUTE_HANDLER : EXCEPTION_CONTINUE_SEARCH)  
     {  
       // code to handle exception  
       puts("Exception Occurred");     
     }  
}    
```  
  
## <a name="see-also"></a>Viz také  
 [try-except – příkaz](http://msdn.microsoft.com/library/30d60071-ea49-4bfb-a8e6-7a420de66381)



